ironfish/src/consensus/verifier.ts

```diff
@@ -296,7 +296,7 @@ export class Verifier {
       return { valid: false, reason: VerificationResultReason.MAX_TRANSACTION_SIZE_EXCEEDED }
     }
 
-    if (!transaction.isMinersFee() && transaction.fee() < consensus.parameters.minFee) {
+    if (transaction.fee() < consensus.parameters.minFee) {
```

Comment:
This was originally only checking min feee if it wasnt a miners fee transaction, which it really should check regardless, since this is checked for new transactions as they come over the network. Miners fee transactions should not ever come over the network, so we can just remove this check.

This is what ultimately replaces the fee check in pool.verify